### ISSUE TITLE
Feature request: A better WER threshold for Deep Speech V2

### ISSUE THREAD

**Issue by d-jax** [tensorflow/models#6374]
### System information
- **What is the top-level directory of the model you are using**:
models/research/deep_speech/
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:
binary
- **TensorFlow version (use command below)**:
tensorflow-gpu==1.12.0
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
9.0
- **GPU model and memory**:
Nvidia Tesla V100
- **Exact command to reproduce**:
run_deep_speech.sh

### Describe the problem
The default WER threshold for training Deep Speech V2 is 23.0, something the author copied from [mlperf](https://github.com/mlperf/training/tree/master/speech_recognition). This is significantly higher than the WER reported by the paper. It is also unclear if the model can reproduce the SotA WER if "unlimited" hardware resource / time can be spent to train it. 

Please consider providing a training configuration that has been confirmed to reproduce the SotA.

**Comment by aoussou** [comment]
@d-jax 

In fact, if I'm not mistaken, a more problematic issue is that WER is not the same as accuracy. The smaller the WER, the better. But if you look at the stopping criterion in [model_helpers.py](https://github.com/tensorflow/models/blob/master/official/utils/misc/model_helpers.py) it is assumed that the measure is something akin to accuracy. 

So if you use [run_deep_speech.sh](https://github.com/tensorflow/models/blob/master/research/deep_speech/run_deep_speech.sh), training almost always stops after the first epoch.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by YuanEric88** [comment]
@aoussou 

I think you are right, it is obviously that the smaller of WER, the better performance that model has. After considering this problem, what is your best WER finally?

**Comment by aoussou** [comment]
> @aoussou
> 
> I think you are right, it is obviously that the smaller of WER, the better performance that model has. After considering this problem, what is your best WER finally?

Sorry, didn't have time to try and fix the problem so I just used another model...

**Comment by YuanEric88** [comment]
@aoussou 
That's totally fine, so what model did you use finally? how is the performance of that?

**Comment by aoussou** [comment]
> @aoussou
> That's totally fine, so what model did you use finally? how is the performance of that?

Check this out: https://github.com/syhw/wer_are_we
